tools/java/common/src/com/google/i18n/phonenumbers/BuildMetadataFromXml.java

```diff
@@ -202,7 +202,7 @@ static String getNationalPrefix(Element element) {
             validateRE(element.getAttribute(NATIONAL_PREFIX_TRANSFORM_RULE)));
       }
     }
-    if (!nationalPrefix.isEmpty()) {
+    if (!nationalPrefix.length() == 0) {
```

Comment:
For consistency, we could change this to check it's not the empty string, like we did for the region code comparison. Or otherwise, change it there to the length check.